contracts/p1/BasketHandler.sol

```diff
@@ -20,6 +20,7 @@ import "./mixins/Component.sol";
 contract BasketHandlerP1 is ComponentP1, IBasketHandler {
     using BasketLibP1 for Basket;
     using EnumerableMap for EnumerableMap.Bytes32ToUintMap;
+    using EnumerableSet for EnumerableSet.Bytes32Set;
```

Comment:
This is a nit on a nit, but we don't need this in BasketHandlerP1, only BasketLib where it already is

Reply:
Yeah, only because it makes it clear for someone reading it. Doesn't change the bytecode anyway.